storage/s3/build.gradle

```diff
@@ -24,11 +24,14 @@ dependencies {
     implementation project(":storage:core")
 
     // Take out jackson libraries out to avoid collision with core jackson dependencies
-    implementation ("software.amazon.awssdk:s3:$awsSdkVersion") {
-        exclude group: "com.fasterxml.jackson.core"
-        exclude group: "com.fasterxml.jackson.dataformat"
-        exclude group: "org.slf4j"
+    def excludeFromAWSDeps = { ModuleDependency dep ->
+        dep.exclude group: "com.fasterxml.jackson.core"
+        dep.exclude group: "com.fasterxml.jackson.dataformat"
```

Comment:
Instead of using this excludes everywhere should we just add
` implementation enforcedPlatform("com.fasterxml.jackson:jackson-bom:$jacksonVersion")`
in subprojects in parent pom?

Reply:
In fact, Jackson doesn't need exclusion, because the AWS libraries has it shadowed.
And SLF4J needs exactly exclusion, because we don't include in on the core as well (expect it to be in Kafka).
Please check the fixup

Reply:
LGTM, however we could replace SLF4J exclusion globally for `runtimeOnly` on configuration level to avoid copy-paste. Will make a PR later. Please squash.

Reply:
Done